dev_docs/tutorials/performance/adding_custom_performance_metrics.mdx

```diff
@@ -181,7 +186,132 @@ Please contact the #kibana-performance team if you need more help visualizing an
 All users who are opted in to report telemetry will start reporting event based telemetry as well.
 The data is available to be analyzed on the production telemetry cluster.
 
+# Report `kibana:plugin_render_time` metric event.
+
+The metric `kibana:plugin_render_time` measures the time from the start of navigation to the point at which the most meaningful component appears on the screen.
```

Comment:
What we want is form the browser navigation start time. I am not sure if we are capturing that, If not we need to change the implementation.

Reply:
The performance mark `start::pageChange` is set on [route change](https://github.com/elastic/kibana/blob/a4f468b97b7981170825c37371676be2696c7f68/x-pack/plugins/observability_solution/apm/public/components/routing/app_root/index.tsx#L74C19-L75C49) and the `kibana:plugin_render_time` is the duration between the `start::pageChange` and the `end::pageReady`.

<img width="1288" alt="Screenshot 2024-05-27 at 12 36 05" src="https://github.com/elastic/kibana/assets/3369346/c29a9dc4-57ee-4120-97e4-be02bd8889f9">


However, I've noticed that there are more than one markers for `start::pageChange` per route and the `performance.measure` gets the latest marker for `start::pageChange`. it seems to be due to re-render. 

https://github.com/elastic/kibana/blob/a4f468b97b7981170825c37371676be2696c7f68/packages/kbn-ebt-tools/src/performance_metrics/context/performance_context.tsx#L41


I believe I need to fix this.

Reply:
There is other way to do this, Where you don't need to set marks on mount start/end. To truly measure based on the navigation time, we could do `performance.measure("render-time)"` this would get the metric between the Nav time and when the page is ready. 

Thanks for checking, it seems as you mentioned we need a followup to fix this.

Reply:
> we could do performance.measure("render-time)"

Where should we do the `performance.measure("render-time)` ? I think I missed this part